include/envoy/event/dispatcher.h

```diff
@@ -59,7 +61,7 @@ class Dispatcher {
    * should only be used on the thread that runs this dispatcher.
    * @return Network::DnsResolverPtr that is owned by the caller.
    */
-  virtual Network::DnsResolverPtr createDnsResolver() PURE;
+  virtual Network::DnsResolverPtr createDnsResolver(std::vector<std::string> resolvers) PURE;
```

Comment:
It probably makes more sense to pass a `const &` to the vector here rather than copy.